portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsMultipleSelectionField.testcase

```diff
@@ -173,13 +173,53 @@ definition {
 	}
 
 	@description = "Verify that options can be rearranged"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test OptionsCanBeRearranged {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-132274 OptionsCanBeRearranged pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Multiple Selection");
+
+		FormFields.editOptionsRow(
+			optionName = "Option1",
+			optionValue = "Option 1",
+			rowNumber = "1");
+
+		FormFields.editOptionsRow(
+			optionName = "Option2",
+			optionValue = "Option 2",
+			rowNumber = "2");
+
+		FormFields.moveGridOptionsRow(
+			key_gridField = "",
+			key_rowNumber1 = "2",
+			key_rowNumber2 = "1");
```

Comment:
@leticiachiu, can we use the macro `moveOptionsRow` instead?

The one chosen by you seems to be specific to the Grid Field.

In addition, I think the step `Assert that "Option 2" is listed above "Option 1" in the Multiple Selection Field` is missing after the drag and drop, according to the Jira ticket. Can you please check it?

Thanks.